### PR TITLE
add opencart screenshots

### PR DESCRIPTION
**Pull request by OnGle**

### CODE REVIEW

File: src/apps/opencart/mod.rs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+use futures::FutureExt;
+use thirtyfour::prelude::*;
+
+pub const APP: App = App {
+    test: &[
+        Step {
+            name: "login",
+            f: |st: &State| {
+                async {
+                    st.goto(st.url.join("index.php?route=account/login&language=en-gb")?.as_str()).await?;
```
Comment (a3s7p) [inline src/apps/opencart/mod.rs]: you don't need to join the path to `st.url` if you use `st.goto`, the function does it

File: src/apps/opencart/mod.rs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                }
+                .boxed()
+            },
+            ..Step::default()
+        },
+        Step {
+            name: "admin-login",
+            f: |st: &State| {
+                async {
+                    st.goto(st.url.join("turnkey_admin")?.as_str()).await?;
```
Comment (a3s7p) [inline src/apps/opencart/mod.rs]: you don't need to join the path to `st.url` if you use `st.goto`, the function does it

File: src/apps/opencart/mod.rs (hunk 1 above)
Comment (a3s7p) [reply src/apps/opencart/mod.rs]: https://github.com/turnkeylinux/clicksnap/blob/master/src/apps/mod.rs#L83

File: src/apps/opencart/mod.rs (hunk 2 above)
Comment (a3s7p) [reply src/apps/opencart/mod.rs]: https://github.com/turnkeylinux/clicksnap/blob/master/src/apps/mod.rs#L83

**Comment by a3s7p** [comment]
otherwise lgtm

**Comment by OnGle** [comment]
Thanks mate I'll try and sort these out today if I get the time.

**Comment by JedMeister** [comment]
Merging now, thanks guys.

**Pull request merged**